packages/client/src/DbAdminApi.ts

```diff
@@ -0,0 +1,26 @@
+import { ClientResponse, RepositoryClient } from "./RepositoryClient.js";
+import { LionwebResponse } from "./responses.js"
+
+export class DbAdminApi {
+    client: RepositoryClient
+    
+    constructor(client: RepositoryClient) {
+        this.client = client
+    }
+
+    async init(): Promise<ClientResponse<LionwebResponse>> {
+        const x = await this.client.postWithTimeout("init", { body: {}, params: "" })
```

Comment:
I would inline all of these calls, removing all of the`x` variables

Reply:
done